### ISSUE TITLE
The identifier next to "async def" is not highlighted as function name

### ISSUE THREAD

**Issue by achimnol** [hdima/python-syntax#36]
..and the fix is:

```diff
diff --git a/syntax/python.vim b/syntax/python.vim
index e461241..4ad66b3 100644
--- a/syntax/python.vim
+++ b/syntax/python.vim
@@ -176,7 +176,7 @@ else
   syn keyword pythonBoolean     True False
   syn match   pythonFunction    "\%([^[:cntrl:][:space:][:punct:][:digit:]]\|_\)\%([^[:cntrl:][:punct:][:space:]]\|_\)*" display contained
   syn keyword pythonStatement   await
-  syn match   pythonStatement   "\<async\s\+def\>" display
+  syn match   pythonStatement   "\<async\s\+def\>" nextgroup=pythonFunction skipwhite
   syn match   pythonStatement   "\<async\s\+with\>" display
   syn match   pythonStatement   "\<async\s\+for\>" display
   syn match   pythonStatement   "\<async\s\+with\>" display```

**Comment by hdima** [comment]
Thank you for the patch! Can you create a pull request?

**Comment by hdima** [comment]
Thank you! I've merged e4d992d
